Source/Extensions/Blazorise.Components/Contexts/AutoCompleteBadgeContext.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Components.Autocomplete;
+
+/// <summary>
+/// Holds the BadgeContext for the current Autocomplete item.
+/// </summary>
+public class BadgeContext<TItem, TValue> : BaseTemplateContext<TItem>
+{
+    /// <summary>
+    /// Initializes BadgeContext.
+    /// </summary>
+    /// <param name="item">Holds current Autocomplete Item Data.</param>
+    /// <param name="value">Holds current Autocomplete Item Value.</param>
+    /// <param name="text">Holds current Autocomplete Item Text.</param>
+    /// <param name="removeCallback">Removes Item from selection callback.</param>
+    public BadgeContext( TItem item, TValue value, string text, EventCallback removeCallback ) : base( item )
```

Comment:
Rename it to `AutocompleteBadgeContext`.

Reply:
I thought about it, but for some reason we didn't do that here:
![image](https://github.com/Megabit/Blazorise/assets/22283161/a75b0051-7094-401d-99a3-9c0571cbdc98)
So I just used the same approach for consistency, because they are both namespace to **Autocomplete** already.
![image](https://github.com/Megabit/Blazorise/assets/22283161/3e2a93d7-ae8a-47ed-9eb2-63511bdabbd2)

Should we still rename anyway?

Reply:
@stsrki friendly reminder

Reply:
Yes, rename it to AutocompleteTagContext.